homeassistant/components/efesto/climate.py

```diff
@@ -0,0 +1,213 @@
+"""Support for Efesto heating devices."""
+import logging
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.climate import ClimateDevice, PLATFORM_SCHEMA
+from homeassistant.components.climate.const import (
+    SUPPORT_ON_OFF, ATTR_OPERATION_MODE, SUPPORT_TARGET_TEMPERATURE
+)
+from homeassistant.const import (
+    ATTR_TEMPERATURE,
+    CONF_URL, CONF_PASSWORD, CONF_USERNAME, CONF_DEVICE,
+    PRECISION_WHOLE, STATE_OFF, STATE_ON, TEMP_CELSIUS
+)
+
+REQUIREMENTS = ['efestoclient==0.0.7']
```

Comment:
Indeed, wasn't aware of this, probably because I've made the component when 0.92 dev was just started and developer documentation probably wasn't updated yet. Thanks!

Reply:
This should no longer be there. Manifest file handles this.